jspdf.js

```diff
@@ -2185,12 +2190,12 @@ var jsPDF = (function(global) {
      * @param  {String} filename The filename including extension.
      *
      * @function
-     * @returns {jsPDF}
+     * @returns {jsPDF} 
      * @methodOf jsPDF#
      * @name save
      */
     API.save = function(filename) {
-      API.output('save', filename);
+      return API.output('save', filename);
```

Comment:
Do you get then true or false as result if it was successful or why did you add the return?